src/views/TabComponent.tsx

```diff
@@ -47,14 +47,39 @@ export class TabComponent extends React.Component<TabProps, TabState> {
 
 export class Tab {
     public sessions: Session[] = [];
+    /*
+    session view map has containing all sessions positions in format:
+    [ 1 4 5], where each number means count of columns in corresponding row
+     */
+    public sessionsViewMap: number[] = [
+      1,
+    ];
```

Comment:
I'm afraid it should be a tree, not a 2d array. For example, if you split a window first vertically and then horizontally in iTerm, the horizontal split will only affect the right pane, whereas with this implementation it's split the whole screen.